spec.html

```diff
@@ -4764,6 +4764,34 @@ <h1>GetFunctionRealm ( _obj_ )</h1>
         <p>Step 5 will only be reached if _obj_ is a non-standard function exotic object that does not have a [[Realm]] internal slot.</p>
       </emu-note>
     </emu-clause>
+
+    <emu-clause id="sec-copydataproperties" aoid="CopyDataProperties">
+      <h1>CopyDataProperties (_target_, _source_, _excluded_)</h1>
+      <p>When the abstract operation CopyDataProperties is called with arguments _target_, _source_ and _excluded_, the following steps are taken:</p>
+      <emu-alg>
+        1. Assert: Type(_target_) is Object.
+        1. Assert: Type(_excluded_) is List.
```

Comment:
`_excluded_` is not an ECMAScript object and so it doesn't have a type. A better wording is, `Assert: _excluded_ is a List of property keys`.

Reply:
> `_excluded_` is not an ECMAScript object and so it doesn't have a type.

Not so: [the definition of Type(x)](https://tc39.github.io/ecma262/#sec-ecmascript-data-types-and-values) specifically allows specification types as well as ECMAScript language types. However, it's true that this capability isn't used much, and never to identify Lists, so I agree with the suggested change.

Reply:
Oh, my mistake! Thanks for the background, but sounds like it's still a reasonable change.